node/node.go

```diff
@@ -1,7 +1,5 @@
 package node
 
 type Node struct {
-	CPUs   int    `json:"cpus" yaml:"cpus"`
-	ID     string `json:"id" yaml:"id"`
-	Memory string `json:"memory" yaml:"memory"`
+	ID string `json:"id" yaml:"id"`
```

Comment:
ID is the node ID. Some identifier to differentiate the different nodes. This is used for instance in the KVM operator to make up configmap names. This might be also of interest for other operators, so I think this is still good here.